src/finance_tracker/tui/app.py

```
@@ -1,7 +1,41 @@
+
+class FinanceTrackerApp(App):
     """Entry point for the `finance-tracker` console script.
 
-    Stub for now (ticket 2 scaffold) — the real Textual `App` and its
+    Placeholder stub for now (ticket 2 scaffold) — the real Textual `App` and its
     placeholder screen land in ticket 8.
     """
-    print("finance-tracker: not yet implemented")
\ No newline at end of file
+    
+    BINDINGS = [("d", "toggle_dark", "Toggle dark mode"), ("ctrl+q", "quit", "Quit")]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)